api/api/middlewares.py

```diff
@@ -7,14 +7,17 @@
 
 from aiohttp import web, web_request
 from aiohttp.web_exceptions import HTTPException
-from connexion.exceptions import ProblemException, OAuthProblem, Unauthorized
+from connexion.exceptions import OAuthProblem, ProblemException, Unauthorized
 from connexion.problem import problem as connexion_problem
 from secure import SecureHeaders
+from wazuh.core.exception import WazuhPermissionError, WazuhTooManyRequests
+from wazuh.core.utils import get_utc_now
 
 from api.configuration import api_conf
 from api.util import raise_if_exc
-from wazuh.core.exception import WazuhTooManyRequests, WazuhPermissionError
-from wazuh.core.utils import get_utc_now
+
+MAX_REQUESTS_DEFAULT = 300
```

Comment:
The default max requests is already defined somewhere else (default config), why do we need to declare another variable?